api/constants/constants.go

```diff
@@ -239,3 +239,11 @@ const (
 	// session recording fails.
 	SessionRecordingModeBestEffort = SessionRecordingMode("best_effort")
 )
+
+// /etc/os-release IDs
+const (
```

Comment:
These constants are not used anywhere
Where are you planning on using them?
Do you think we can remove them and add them back later on when we add the `os-release ids` for "compatible" distros in the AWS Server discover?

We can use them in the tests, but I don't think it should be its main purpose